lib/backup_restore/restorer.rb

```diff
@@ -106,10 +106,19 @@ def initialize_state
       @current_version = BackupRestore.current_version
       @timestamp = Time.now.strftime("%Y-%m-%d-%H%M%S")
       @tmp_directory = File.join(Rails.root, "tmp", "restores", @current_db, @timestamp)
+      @source_filename = File.join(Backup.base_directory, @filename)
       @archive_filename = File.join(@tmp_directory, @filename)
       @tar_filename = @archive_filename[0...-3]
       @meta_filename = File.join(@tmp_directory, BackupRestore::METADATA_FILE)
-      @dump_filename = File.join(@tmp_directory, BackupRestore::DUMP_FILE)
+
+      # For backwards compatibility
+      @dump_filename =
+        if system("tar --list --verbose --file #{@source_filename} #{BackupRestore::DUMP_FILE}")
```

Comment:
Do we need to do a `--verbose` when we only care about the exit status of the command?